docs/manual-EN/3.build-develop-and-administration/1.build/1.build-source-code.md

```diff
@@ -2,7 +2,7 @@
 
 ## Prerequisite
 
-Nebula is developed based C++14, so it requires a compiler supporting C++14 features.
+Nebula Graph is developed based C++14, so it requires a compiler supporting C++14 features.
 
 ### Supported system version
```

Comment:
Please check all the words in headings

Reply:
Will open a new pr to take care of all the headings.